packages/xrpl/test/integration/transactions/credentialDelete.test.ts

```
@@ -0,0 +1,88 @@
+      Issuer: testContext.wallet.classicAddress,
+      CredentialType: stringToHex('Test Credential Type'),
+    }
+
+    await testTransaction(testContext.client, credentialDeleteTx, subjectWallet)
+
+    // Check both issuer and subject no longer have a credential tied to the account
+    const SubjectAccountObjectsDeleteResponse: AccountObjectsResponse =
+      await testContext.client.request({
+        command: 'account_objects',
+        account: subjectWallet.classicAddress,
+        type: 'credential',
```

Biome (1.9.4)

[error] This property value named type is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named type


(lint/suspicious/noDuplicateObjectKeys)